### ISSUE TITLE
Archive and "publish" the course 

### ISSUE THREAD

**Issue by ethanwhite** [datacarpentry/semester-biology#579]
At the end of the semester we should clean up the semester specific tweaks to the schedule and then archive/"publish" the course on Zenodo (or somewhere else that I haven't thought of).

**Comment by brymz** [comment]
Are there open issues or other tasks that need to be completed before archiving the version?

**Comment by brymz** [comment]
- [ ] Update 'Praise' page (#378)
- [ ] Determine name use for customization (#588)

**Comment by ethanwhite** [comment]
I don't think there's anything that I'd consider a blocker for archiving.

**Comment by brymz** [comment]
As zenodo.org allows us to log-in and archive by Github account, do we want to do this through @weecology? I'm happy to proceed with this if you don't mind it hooked to @brymz.

**Comment by ethanwhite** [comment]
I think through @datacarpentry is probably best. I went ahead and flipped the switch so the next release should go automatically. We may need to adjust some things after that happens, but we'll deal with that once we need to.

**Comment by brymz** [comment]
I have generated a draft release that is ready for review/publishing: https://github.com/datacarpentry/semester-biology/releases

**Comment by ethanwhite** [comment]
Done. Sorry for the delay. New issue opened to discuss authorship issues (which I'd totally forgotten about) #617.
